sandbox

```diff
@@ -373,12 +382,28 @@ sandbox () {
         dc up
```

Comment:
Inside of this `if` conditional, we only have `dc up`. I wasn't sure if you guys wanted to run this new logic when the user tries **interactive mode** or not.